src/AppBundle/Controller/BlogController.php

```diff
@@ -126,4 +127,25 @@ public function commentFormAction(Post $post)
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/rss.xml", name="blog_rss")
+     * @Method("GET")
+     */
+    public function rssAction()
```

Comment:
@yceruto, I think it's not needed to create a particular action. You could use the `_format` parameter in the `indexAction`. See http://symfony.com/doc/current/quick_tour/the_controller.html#using-formats